srcpkgs/justbuild/template

```diff
@@ -0,0 +1,68 @@
+# Template file for 'justbuild'
+pkgname=justbuild
+version=1.1.2
+revision=1
+create_wrksrc=yes
+hostmakedepends="jo pandoc pkg-config python3 tar unzip wget"
+makedepends="abseil-cpp-devel c-ares-devel catch2 fmt-devel grpc-devel json-c++
+ libarchive-devel libcurl-devel libgit2-devel openssl-devel re2-devel"
+depends="python3"
+short_desc="Generic build system supporting multi-repository builds"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
+license="Apache-2.0"
+homepage="https://github.com/just-buildsystem/justbuild/"
+changelog="https://github.com/just-buildsystem/justbuild/blob/master/CHANGELOG.md"
+distfiles="https://github.com/just-buildsystem/justbuild/archive/v${version}.tar.gz"
+checksum=50341bdf3742ad6dec20454b338770e8428b531d04c675ca976d1e410c5b33ba
+conflicts="just>=0"
+nocross="not supported by bootstrapping"
+
+do_build() {
+	mkdir -p just-work
+	export JUST_BUILD_CONF=$(jo \
+		AR=/usr/bin/ar \
+		COMPILER_FAMILY=gnu \
+		CC=$CC \
+		CXX=$CXX \
+		CFLAGS=$(jo -a -- $CFLAGS) \
+		CXXFLAGS=$(jo -a -- $CXXFLAGS) \
+		LDFLAGS=$(jo -a -- $LDFLAGS) \
+		SOURCE_DATE_EPOCH=$SOURCE_DATE_EPOCH \
+		VERSION_EXTRA_SUFFIX=-void
+	)
+	export NON_LOCAL_DEPS=$(jo -a \
+		bazel_remote_apis \
+		cli11 \
+		com_github_microsoft_gsl \
+		google_apis \
+		protobuf
+	)
+
+	PACKAGE=YES LOCALBASE=/usr python3 \
+		justbuild-${version}/bin/bootstrap.py \
+		justbuild-${version}/ $PWD/just-work
+
+	./just-work/src/out-boot/bin/just install \
+		-C $PWD/just-work/repo-conf.json \
+		-D "$JUST_BUILD_CONF" \
+		-o $PWD/just-work/out \
+		--local-build-root $PWD/just-work/.just \
+		'' just-mr
+
+	for m in justbuild-${version}/share/man/*.md; do
+		pandoc --standalone --to man -o ${m%.md}.man $m
+	done
+}
+
+do_install() {
+	vbin just-work/out/bin/just
```

Comment:
Maybe worth renaming to `justbuild` since this and https://github.com/casey/just are both active projects.